cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -71,6 +70,7 @@ Scenario Outline: Component <component> basic story
     | step-sequence |
     | icon          |
     | button toggle |
+    | preview       |
```

Comment:
It's a wrong TC for `preview` component. We don't have just `basic` story. It's inside of `test` folder. So it should be a new TC like
 ```  
@accessibility
  Scenario Outline: Component <component> test basic page
    When I open "<component> Test" component page "basic" in no iframe
    Then "<component>" component has no accessibility violations
    Examples:
      | component        |
      | preview          |

```